[2023-12-16 13:00:01,874] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-16T12:00:00+00:00 [queued]>
[2023-12-16 13:00:01,898] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-16T12:00:00+00:00 [queued]>
[2023-12-16 13:00:01,899] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-12-16 13:00:01,899] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2023-12-16 13:00:01,900] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-12-16 13:00:01,911] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): fill> on 2023-12-16T12:00:00+00:00
[2023-12-16 13:00:01,918] {standard_task_runner.py:52} INFO - Started process 5304 to run task
[2023-12-16 13:00:01,923] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_python_weather_v01', 'fill', '2023-12-16T12:00:00+00:00', '--job-id', '103', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_weather.py', '--cfg-path', '/tmp/tmpnej1stb2', '--error-file', '/tmp/tmpsd1gaxrx']
[2023-12-16 13:00:01,927] {standard_task_runner.py:77} INFO - Job 103: Subtask fill
[2023-12-16 13:00:01,985] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_python_weather_v01.fill 2023-12-16T12:00:00+00:00 [running]> on host 5f3722f1fc3b
[2023-12-16 13:00:02,038] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ggoni
AIRFLOW_CTX_DAG_ID=dag_with_python_weather_v01
AIRFLOW_CTX_TASK_ID=fill
AIRFLOW_CTX_EXECUTION_DATE=2023-12-16T12:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-16T12:00:00+00:00
[2023-12-16 13:00:05,090] {python.py:118} INFO - Done. Returned value was: None
[2023-12-16 13:00:05,108] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=dag_with_python_weather_v01, task_id=fill, execution_date=20231216T120000, start_date=20231216T130001, end_date=20231216T130005
[2023-12-16 13:00:05,141] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-16 13:00:05,178] {local_task_job.py:146} INFO - Task exited with return code 0
